yoti-sdk-impl/src/main/java/com/yoti/api/client/shareurl/policy/SimpleDynamicPolicyBuilder.java

```diff
@@ -200,6 +202,8 @@ public DynamicPolicyBuilder withPinAuthorisation() {
     public DynamicPolicyBuilder withPinAuthorisation(boolean enabled) {
         if (enabled) {
             return withWantedAuthType(PIN_AUTH_TYPE);
+        } else {
+            this.wantedAuthTypes.remove(PIN_AUTH_TYPE);
```

Comment:
Can we move this into `withWantedAuthType()` as per _Node_ and _.NET_ _(in a non-breaking way)_? This will allow any auth type to be added/removed